.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).
+
+**Description**
+
+> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.
+
+> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.
+
+**Main changes in the PR:**
+
+1. <Change 1>
+2. <Change 2>
+
+**Validation Steps:**
+
+1. <Step 1>
+2. <Step 2>
```

Comment:
Maybe this is just a preference thing but it feels weird to have this framed as "steps." Consider something like "Validation performed:"